src/sample-app/app.component.html

```diff
@@ -62,6 +62,56 @@
                     <p>Content for Cloud tab.</p>
                 </clr-tab-content>
             </clr-tabs>
+            <div class="card card-block">
+                <p class="card-text username-list">
+                    Selected users:
+                    <em *ngIf="selected.length == 0">No user selected.</em>
+                    <span class="username" *ngFor="let user of selected">{{user.name}}</span>
+                </p>
+
+                <p class="card-text username-list">
+                    Users to be added to group:
+                    <em *ngIf="toAdd.length == 0">No user selected.</em>
+                    <span class="username" *ngFor="let user of toAdd">{{user.name}}</span>
+                </p>
+
+                <p class="card-text username-list">
+                    Users to be deleted:
+                    <em *ngIf="toDelete.length == 0">No user selected.</em>
+                    <span class="username" *ngFor="let user of toDelete">{{user.name}}</span>
+                </p>
+
+                <p class="card-text username-list">
+                    User to be edited:
+                    <em *ngIf="!toEdit">No user selected.</em>
+                    <span class="username" *ngIf="toEdit">{{toEdit.name}}</span>
+                </p>
+            </div>
+
+            <clr-datagrid [(clrDgSelected)]="selected">
+                <clr-dg-action-bar>
+                    <div class="btn-group" role="group" aria-label="Available Actions">
+                        <button type="button" class="btn btn-sm btn-secondary" (click)="onAdd()"><clr-icon shape="plus"></clr-icon> Add
+                            to
+                            group</button>
+                        <button type="button" class="btn btn-sm btn-secondary" (click)="onDelete()" ><clr-icon shape="close"></clr-icon> Delete</button>
+                        <button type="button" class="btn btn-sm btn-secondary" (click)="onEdit()" *ngIf="selected.length == 1"><clr-icon shape="pencil"></clr-icon> Edit</button>
+                    </div>
+                </clr-dg-action-bar>
+
+                <clr-dg-column>Name</clr-dg-column>
+
+                <clr-dg-row *clrDgItems="let user of users" [clrDgItem]="user">
+                    <clr-dg-action-overflow>
+                        <a class="action-item" href="#" (click)="onEdit()">Edit</a>
+                        <a class="action-item" href="#" (click)="onDelete()">Delete</a>
+                    </clr-dg-action-overflow>
+
+                    <clr-dg-cell>{{user.name}}</clr-dg-cell>
+                </clr-dg-row>
+
+                <clr-dg-footer>{{users.length}} users</clr-dg-footer>
+            </clr-datagrid>
```

Comment:
Yeah I think it's loose at this point, as this file doesn't impact anything in our build pipeline. I wanted to keep in the datagrid example there, since it's one of the more complex ones that we'd probably need to keep testing as we add more features to it.